tailwind.config.cjs

```diff
@@ -1,8 +1,38 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
+  mode: "jit",
   content: ["./src/**/*.{js,ts,jsx,tsx}"],
+  purge: ["./src/**/*.{js,ts,jsx,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        primary: "#84B0BA",
+        secondary: "#AE4646",
+        tertiary: "#822e2e",
+        white: "#FFFFFF",
+      },
+      fontSize: {
+        3.2: "3.2rem",
+        1.2: "1.2rem",
+      },
+      spacing: {
+        2.4: "2.4rem",
+        3.1: "3.1rem",
+        3.4: "3.4rem",
+        "gap-9-2": "9.2rem",
+      },
+      height: {
+        article: "19rem",
+        hero: "calc(100vh - 23.8rem)",
+      },
+      minHeight: {
+        s: "30rem",
+      },
+      fontFamily: {
+        inter: ["Inter", "sans-serif"],
+      },
```

Comment:
I believe this should not be here and rather be used for all elements in the CSS reset. Something like this

![CleanShot 2023-04-04 at 17 40 06](https://user-images.githubusercontent.com/12488648/229845111-88186da6-711b-42cb-ad92-f7b7496c484d.png)

Or we can do the tailwind approach - https://tailwindcss.com/docs/font-family#customizing-the-default-font

Choose whichever you like more. The first one is more general (people without tailwind knowledge will understand that more likely)

Second one is TW way